Assets/Scripts/ItemPickup.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;

```

Comment:
Zijn deze allemaal nodig?
```suggestion
using UnityEngine;
```
Dit is genoeg.